### DISCUSSION TITLE
In the metadata record, should `subject` be a list?

### DISCUSSION THREAD

**Discussion by dyf** [AllenNeuralDynamics/aind-data-schema#562] [Ideas]
Likely we will be doing multi-subject behavioral experiments. 

If so, how do we name sessions? Some sort of a cohort identifier?

@saskiad

**Comment by saskiad** [comment]
Are there in fact plans to do multi-subject behavior experiments? I haven't heard of that. I'd suggest we wait until the science points us in that direction. But if this is something people are planning, we can make that a list. And we'd have to think of an identifier - might just have to not have subject/cohort identifier for those experiments and just go with modality & date
